spotfinder/CMakeLists.txt

```diff
@@ -22,5 +28,4 @@ target_link_libraries(spotfinder
     nlohmann_json::nlohmann_json
 )
 target_compile_options(spotfinder PRIVATE "$<$<AND:$<CONFIG:Debug>,$<COMPILE_LANGUAGE:CUDA>>:-G>")
-
-
+include_directories(./kernels)
```

Comment:
Is this necessary at all? It looks like the only places you `#include` you do as `#include "kernels/masking.cuh"`, which would be picked up explicitly relative.

In general, `include_directories` adds the directory universally to all targets, which should only be done as a last resort. If a specific target needs to reference something not in a standard findable place, then `target_include_directories` should be considered first (or using the transitive dependencies system).

Reply:
Right, I see what you mean. That was an unnecessary inclusion 😅